src/contexts/todoContext.tsx

```diff
@@ -44,10 +44,10 @@ const reducer = (state: TodoState, action: Action): TodoState => {
         // todos:
       };
     case 'DELETE_TODO':
+      api.deleteTodo(action.id);
       return {
         ...state,
-        // TODO: todo가 삭제 되었을 때 action.id 값을 이용하여 todos 가 삭제 되도록 수정하기
-        // todos:
+        todos: api.getTodos(),
```

Comment:
[이 코드](https://github.com/EveryAnalytics-Study/todo-list/blob/main/src/api/todo.ts#L66)에서 보다싶이 api.deleteTodo 함수는 삭제된 결과의 배열을 return 값으로 반환하고 있기 때문에,
굳이 api.getTodos() 호출할 필요 없이 아래처럼 바꾸면 됩니다!

```suggestion
case 'DELETE_TODO':
    return {
        ...state,
        todos: api.deleteTodo(action.id),
        };
```